docker/Dockerfile

```diff
@@ -122,26 +112,17 @@ ENV NVIDIA_VISIBLE_DEVICES="all"
 ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
 
 # Install dependencies:
-# curl: healthcheck
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests --yes \
-    ca-certificates \
-    gnupg \
-    curl \
-    wget \
-    apt-transport-https \
+    ca-certificates gnupg curl \
```

Comment:
`apt-transport-https` is definitely needed here, since the repo is TLS.

Also, as above.